opm/simulators/flow/ActionHandler.hpp

```diff
@@ -31,19 +31,20 @@
 #include <unordered_map>
 #include <vector>
 
-namespace Opm {
-
-namespace Action {
-class ActionX;
-class State;
-}
+namespace Opm::Action {
+    class State;
+} // namespace Opm::Action
 
+namespace Opm {
 template<class Scalar> class BlackoilWellModelGeneric;
 class EclipseState;
 class Schedule;
 struct SimulatorUpdate;
 class SummaryState;
 class UDQState;
+} // maespace Opm
```

Comment:
namespace

Reply:
> namespace

Good catch.  Wow&ndash;that's a truly messed up spelling :fearful: